src/encord_active/frontend/src/components/preview/AnnotatedImage.tsx

```diff
@@ -99,7 +99,7 @@ export function AnnotatedImage(props: {
       ) : (
         <img
           ref={image}
-          className="!rounded-none object-contain transition-opacity"
```

Comment:
I think that this may be the cause of the de-sync between annotations and images when being displayed. The svg positioning & size metadata needs to be checked.